src/problem_bank_scripts/problem_bank_scripts.py

```diff
@@ -1066,6 +1089,13 @@ def process_question_pl(source_filepath, output_path=None):
         pl_path.mkdir(parents=True, exist_ok=True)
         [copy2(pathlib.Path(source_filepath).parent / fl, pl_path / fl) for fl in files_to_copy]
 
+    # Move autograde py test files
+    files_to_copy = parsed_q["header"].get("autogradeTestFiles")
+    if files_to_copy:
+        pl_path = output_path / "tests"
```

Comment:
It's important to troubleshoot and test this output_path business...it's very finicky.